gensim/models/keyedvectors.py

```diff
@@ -1902,3 +1884,18 @@ def pseudorandom_weak_vector(size, seed_string=None, hashfxn=hash):
     else:
         once = utils.default_prng
     return (once.random(size).astype(REAL) - 0.5) / size
+
+
+def prep_vectors(target_shape, prior_vectors=None, seed=0, dtype=REAL):
+    """Return a numpy array of the given shape. Reuse prior_vectors values instance or values
+    to extent possible. Initialize new values randomly if requested."""
```

Comment:
This seems to be a public API function. We should document the parameters so they appear in the documentation.

Reply:
It ultimately may not be, pending the mmap work & other initialization clean-up, which might also jostle the internal names a bit. (At the moment, this is *only* called from `resize_vectors()` which may be the preferable public-entry point, because outside callers are less-likely to have a `prior_vectors`.)

Reply:
OK, but this stuff will show up in the docs, right?

I can think of several better ways forward:

- Make the docstring a code-comment so it doesn't show up in the docs
- Mark the function as internal